docs/index.rst

```diff
@@ -86,126 +80,70 @@ Run demo
 Playbook
 ========
 
-|c3| Tour
---------------------------------
+Brokers 
+-------
 
-Follow along with the `Demo 2: Tour <https://youtu.be/D9nzAxxIv7A>`_ video.
+1. Select the cluster called "Kafka Raleigh".
 
-.. raw:: html
+   .. figure:: images/cluster_raleigh.png
 
-    <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
-        <iframe src="https://www.youtube.com/embed/D9nzAxxIv7A" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 75%; height: 75%;"></iframe>
-    </div>
+2. Click on "Brokers".
 
-1. **MONITORING –> System Health**: |c3| landing
-   page shows the overall `system
-   health <https://docs.confluent.io/current/control-center/docs/systemhealth.html>`__
-   of a given Kafka cluster. For capacity planning activities, view
-   cluster utilization:
+3. View the status of the Brokers in the cluster, including:
 
-   -  CPU: look at network and thread pool usage, produce and fetch
-      request latencies
-   -  Network utilization: look at throughput per broker or per cluster
-   -  Disk utilization: look at disk space used by all log segments, per
-      broker
+   - Produce metrics
+   - Consume metrics
+   - Broker uptime
+   - Partitions: online, under replicated, total replicas, out of sync replicas
+   - Disk utilization
+   - System: network pool usage, request pool usage
 
    .. figure:: images/landing_page.png
 
+Topics
+------
 
+1. |c3| has a useful interface to manage topics in a Kafka cluster. Click on "Topics".
 
+2. Scroll down to and click on the topic `wikipedia.parsed`.
 
-2. **MANAGEMENT –> Kafka Connect**: |c3| uses
-   the Kafka Connect API to manage `Kafka
-   connectors <https://docs.confluent.io/current/control-center/docs/connect.html>`__.
-
-   -  Kafka Connect **Sources** tab shows the connectors
-      ``wikipedia-irc`` and ``replicate-topic``. Click ``Edit`` to see
-      the details of the connector configuration and custom transforms.
-
-      .. figure:: images/connect_source.png
-         :alt: image
-   
-
-
-
-   -  Kafka Connect **Sinks** tab shows the connector
-      ``elasticsearch-ksql``. Click ``Edit`` to see the details of the
-      connector configuration and custom transforms.
-
-      .. figure:: images/connect_sink.png
+      .. figure:: images/topic_list_wikipedia.png
          :alt: image
 
+3. View an overview of this topic:
 
+   - Throughput
+   - Partitions: under replicated, out of sync replicas
```

Comment:
Just wording suggestions, feel free to ignore any of these you don't agree with...
```suggestion
   - Partition replication status
```